### PR TITLE
Feature/dx 431 tool displays lti launch jwt

### PR DESCRIPTION
**Pull request by Eddyjim**

### CODE REVIEW

**Review by shurrey — APPROVED** [review]
Yuck. React. haha Nice job Eddy

**Review by johnMateusBb — APPROVED** [review]

File: src/config/config.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import _ from "lodash";
+import configJson from "./config.json"
+
+let configuration = {};
+
+let configJsonOverride = null;
+
+
+if (process.env.CONFIG_OVERRIDE) {
+    try {
```
Comment (ewpreston) [inline src/config/config.js:8]: I'd like it if we can use 2-space indents everywhere

File: src/config/config.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+if (configJsonOverride) {
+    configuration = _.defaultsDeep(configJsonOverride, configJson);
+}
+
+if (process.env.LTI_BACKEND) {
+    configuration['lti_backend'] = process.env.LTI_BACKEND
+}
+
+
```
Comment (ewpreston) [inline src/config/config.js]: and avoid extra blank lines; The Ultra team does some pretty tight linting and we may want to do that for this project as well

File: src/pages/home/index.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (ewpreston) [inline src/pages/home/index.css:1]: I'm neither a React or CSS expert but Brian Burdan suggested we look at jss for React apps.

File: src/pages/lti/deepLinkContent.js [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+                Request
+            </Button>
+        );
+    } else {
+        return (
+            <Card>
+                <Card.Body>
+                    <JsonAccordion json={items} title={'Items'}/>
+                    <Button onClick={getLTIContent}>
+                        Update
```
Comment (ewpreston) [inline src/pages/lti/deepLinkContent.js]: We should think about localizing this app

**Review by ewpreston — APPROVED** [review]

File: src/pages/home/index.css (hunk 3 above)
Comment (Eddyjim) [reply src/pages/home/index.css:1]: I just changed this

File: src/pages/lti/deepLinkContent.js (hunk 4 above)
Comment (Eddyjim) [reply src/pages/lti/deepLinkContent.js]: I just added a localization functionality

**Pull request merged**